miniapp/src/main/java/com/rakuten/tech/mobile/miniapp/display/MiniAppWebView.kt

```diff
@@ -94,6 +97,18 @@ internal class MiniAppWebView(
         destroy()
     }
 
+    @VisibleForTesting
+    internal fun setupMiniAppNavigator() {
+        if (miniAppNavigator == null) {
```

Comment:
I agree that we should omit `MiniApp.create(appId: String, miniAppMessageBridge: MiniAppMessageBridge)`.

I have updated the following with `null` as the default value for `miniAppNavigator`.
```
abstract suspend fun create(
        appId: String,
        miniAppMessageBridge: MiniAppMessageBridge,
        miniAppNavigator: MiniAppNavigator? = null
): MiniAppDisplay
```

so I think we can omit immediately and no need to change to deprecated status.

Reply:
I think, we can keep `create(String, MiniAppMessageBridge)` and marked it as just deprecated instead of directly delete it in earlier. It will be nicer to give a deprecation warning if it's possible. Also, need to describe this kind thing in CHANGELOG.

Reply:
The users do not see the change. The implementation of default null and `create(String, MiniAppMessageBridge)` are the same. That's why I think there is no need for deprecation warning because there is no deprecated method.